lib/active_admin.rb

```diff
@@ -56,6 +56,9 @@ module ActiveAdmin
   autoload :ViewFactory,              'active_admin/view_factory'
   autoload :ViewHelpers,              'active_admin/view_helpers'
   autoload :Views,                    'active_admin/views'
+  autoload :ExceptionController,      'active_admin/exception_app'
+  autoload :ErrorHandler,             'active_admin/middleware/error_handler'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.